web_security_dialog/LICENSE/LICENSE

```diff
@@ -0,0 +1,17 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+    Copyright (C) 2011-2017 Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
+
```

Comment:
Dear Vishal, please do the push logically.
remove openerp Sa line. Was this module copied by you or made by you?